test_tracking.ini

```diff
@@ -0,0 +1,46 @@
+# OpenFisca-Web-API - Development environment configuration
```

Comment:
If we don't keep the performance test, I'm not sure adding a new config file is worth it just for a tiny test.